### ISSUE TITLE
Missing deps for various X11-related easyconfigs

### ISSUE THREAD

**Issue by verdurin** [hpcugent/easybuild-easyconfigs#4028]
Error when compiling for `foss-2016b`:
```
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for PTHREADSTUBS... no
configure: error: Package requirements (pthread-stubs) were not met:

No package 'pthread-stubs' found
                                                                                                                                                                                                                                                                                                             Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables PTHREADSTUBS_CFLAGS
and PTHREADSTUBS_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
 (at easybuild/tools/run.py:438 in parse_cmd_output)
```

**Comment by verdurin** [comment]
Having added `libphtread-stubs`, I've now found it's missing `pciaccess`:
```
checking for native atomic primitives... Intel
checking for PCIACCESS... no
configure: error: Package requirements (pciaccess >= 0.10) were not met:

No package 'pciaccess' found                                                                                                                                                                                        
Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables PCIACCESS_CFLAGS                                                                                                                                              and PCIACCESS_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
 (at easybuild/tools/run.py:438 in parse_cmd_output)
```

**Comment by boegel** [comment]
@verdurin latter can be fixed by including `libpciaccess` as a dep?

If that resolves the issue, please open a PR to add the missing deps to `libdrm`?

We clearly need a better way of testing things in isolation...

**Comment by verdurin** [comment]
@boegel yes, I've already tested that fix. Will be sending that and several other similar PRs...

**Comment by boegel** [comment]
fixed in #4032
